### PR TITLE
Show ref link on dashboard instead of ref code

### LINKED ISSUE #792: as an affiliate, I'd like to have an easy way to get my affiliate link in the lobby
Current situation:
image

Potential problems:
- it is not clear what this is all about. what benefits will you get? 
- no call to action. what to do next?
- no referral link to share. Should be easy to copy and (optional) directly share on social networks?

Potential solution:
- display referral link so it is easy to copy
- stimulate users to share the link (a-la "Share your personal link and get up to 5% of the deposit your referals bring in")

### LINKED ISSUE #810: Overview Tab: Promo Codes -> No incentive to invite users
There are promo codes displayed, but as a User I don't know why I should invite users.

### PR DESCRIPTION
**Pull request by sunify**
Resolves #792
Resolves #810

image

### CODE REVIEW

File: app/components/Dashboard/Refs.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <SectionOverview
+      name="refs"
+      style={{
+        alignItems: 'center',
+      }}
+    >
+      <H2><FormattedMessage {...messages.refs} values={{ refsCount: refs.length }} /></H2>
+      <p style={{ textAlign: 'center', margin: '-10px 0 20px' }}>
+        Share your personal link and get up to 5%<br />of the deposit your referals bring in (coming soon)
```
Comment (troggy) [inline app/components/Dashboard/Refs.js]: typo: refer**r**als

File: app/components/Dashboard/LinkInput.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import PropTypes from 'prop-types';
+import Input from '../Input';
+
+import { CopyIcon } from './styles';
+
+function copyLink(el) {
+  el.select();
+
+  try {
+    document.execCommand('copy');
```
Comment (troggy) [inline app/components/Dashboard/LinkInput.js]: woah! neat

**Comment by m-kroener** [comment]
Hey, looks good, huge improvement :)
Is it possible that we write sth like ... Earn up to 5% of your referrals first deposit. Payouts will be processed soon. 

I thought it may be better because then there is already an incentive to share, o ly the payout will come soon. But the question is if that is technically possible?

**Comment by johannbarbie** [comment]
i like it too. wasn't the referral bonus only for ico investment? that is not happening any time soon though.

**Comment by sunify** [comment]
@m-kroener @johannbarbie what about wording? Let's pick some up and merge the PR :-)

**Comment by johannbarbie** [comment]
we currently can't track first deposits well, and also it would be risky to pay bonuses based on those, as we could be easily gamed.
currently there is no bonus on referrals other then part of the basic income.
you can say: 
"share your personal link and get up to 5% of your referrals first powerUp."
not saying when powerUps will be enabled or anything.

**Comment by m-kroener** [comment]
Share your personal link and get up to 5% bonus of your referrals first powerUp"

I am good with it :)

**Pull request merged**